trigger/testTimer/README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+title: CSV Timer
+
+-->
+# Timer Trigger
+This trigger provides your flogo application the ability to get data from a CSV file after a particular interval for an action 
+
+## Installation
+
+```bash
+flogo install github.com/skothari-tibco/csvtrigger
+```
+
+## Metadata
+```json
+{
+  "handler": {
+    "settings": [
+      {
+        "name": "filePath",
+        "type": "string"
+      },
+      {
+        "name": "repeatInterval",
+        "type": "string"
+      }
+    ]
+  }
+}
+```
+### Details
+
+#### Settings:
+| Setting  | Required | Description |
+|:---------|:---------|:------------|
+| control  | false    | Enable control of trigger
```

Comment:
I would remove this, the control should always be enabled, You could consider an autostart flag instead